From 1815cfbb56daf4a4054044cf5db0dddf1e2f4dd3 Mon Sep 17 00:00:00 2001
From: Gaurav Kumar Garg <garg.gaurav52@gmail.com>
Date: Wed, 12 Aug 2015 01:09:41 +0530
Subject: [PATCH 61/80] glusterd: DEBUG log should not come after resetting client log level

This patch is backport of: http://review.gluster.org/#/c/11887/

After resetting diagnostics.client-log-level option still DEBUG log is
logging in scrubber and bitrot log file. After resetting any option
value of that options goes to default value.

This patch will set the default value of client and brick log level to
"INFO" log level.

GlusterFS release 3.7.9 patch url: http://review.gluster.org/#/c/13640/

  >> Change-Id: I4cf04754dcf5ddc908dae4d9bdf525cfcd6cc2bd
  >> BUG: 1252696
  >> Signed-off-by: Gaurav Kumar Garg <ggarg@redhat.com>
  >> Reviewed-on: http://review.gluster.org/11887
  >> Tested-by: NetBSD Build System <jenkins@build.gluster.org>
  >> Tested-by: Gluster Build System <jenkins@build.gluster.com>
  >> Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
(cherry picked from commit 6e17fb2097f941798e1d56728fd3d61e525a39a5)

Change-Id: I95b89db25de14cb32f8b392ca2b7ac3531617708
BUG: 1231150
Signed-off-by: Gaurav Kumar Garg <ggarg@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/71848
Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
Tested-by: Atin Mukherjee <amukherj@redhat.com>
---
 xlators/mgmt/glusterd/src/glusterd-volume-set.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/xlators/mgmt/glusterd/src/glusterd-volume-set.c b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
index 7bb8f91..0381dd1 100644
--- a/xlators/mgmt/glusterd/src/glusterd-volume-set.c
+++ b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
@@ -1171,11 +1171,13 @@ struct volopt_map_entry glusterd_volopt_map[] = {
         },
         { .key         = "diagnostics.brick-log-level",
           .voltype     = "debug/io-stats",
+          .value       = "INFO",
           .option      = "!brick-log-level",
           .op_version  = 1
         },
         { .key        = "diagnostics.client-log-level",
           .voltype    = "debug/io-stats",
+          .value      = "INFO",
           .option     = "!client-log-level",
           .op_version = 1,
           .flags      = OPT_FLAG_CLIENT_OPT
-- 
1.7.1

